src/clientRoutes.js

```diff
@@ -572,6 +580,8 @@ exports = module.exports = function(app, args) {
 
   app.post('/api/v2/:coin/wallet/:id/sweep', parseBody, prepareBitGo(args), promiseWrapper(handleV2Sweep, args));
 
+  // CPFP
+  app.post('/api/v2/:coin/wallet/:id/bump', parseBody, prepareBitGo(args), promiseWrapper(handleV2BumpTransaction, args));
```

Comment:
with `accelerateTransaction` as the func name we should call this `acceleratetx` (see `signtx`)

Reply:
done